syntax = "proto3";

package router;

import "google/protobuf/empty.proto";

service Router {
    rpc RegisterNode (RegisterNodeRequest) returns (google.protobuf.Empty) {};
}

message RegisterNodeRequest {
    string server_address = 1;
}
